xla/service/cpu_gpu_shape_verifier.cc

```diff
@@ -75,7 +76,9 @@ absl::Status VerifyS4U4Usage(HloInstruction* instruction) {
     case HloOpcode::kDynamicUpdateSlice:
     case HloOpcode::kFusion:
     case HloOpcode::kGetTupleElement:
+    case HloOpcode::kOptimizationBarrier:
     case HloOpcode::kParameter:
+    case HloOpcode::kSelect:
```

Comment:
Why is `select` here? It's an elementwise, right?

Reply:
Indeed, same as kBitcastConvert. Removed both.